corpus/decl/def.nu

```diff
@@ -110,3 +110,199 @@ def-env test []: nothing -> record<a: string, b: int> {}
         (identifier)
         (flat_type)))
     (block)))
+
+======
+def-007-long-flag
+======
+
+def test [
+  --long-flag
+] {}
+
+-----
+
+(nu_script
+  (decl_def
+    (cmd_identifier)
+    (parameter_bracks
+      (parameter
+        (param_long_flag
+          (identifier))))
+    (block)))
+
+======
+def-008-long-flag-with-short
+======
+
+def test [
+  --long-flag (-l)
```

Comment:
does this require a space or will it parse like `--long-flag(-l)`?

Reply:
A space is not required.